app/Resources/views/full/home_page.html.twig

```diff
@@ -46,14 +42,10 @@
 
             <div class="row space-bottom-less">
                 <div class="col-md-10 col-md-offset-1">
-                    {% if blogPosts|length > 0 %}
-                        {% for post in blogPosts %}
-                            {{ render_esi(controller('ez_content:viewLocation', {
-                                locationId: post.contentInfo.mainLocationId,
-                                viewType: 'home'
-                            })) }}
-                        {% endfor %}
-                    {% endif %}
+                    {{ render_esi(controller('ez_content:viewLocation', {
```

Comment:
Reminder: viewLocation is deprecated. You should use viewContent, with both `contentId` and `locationId` as arguments.